client/my-sites/email/new-mailbox-list/constants.ts

```diff
@@ -0,0 +1,19 @@
+/**
+ * Field names.
+ * Used in forms to identify common fields across providers
+ */
+export const FIELD_ALTERNATIVE_EMAIL = 'alternativeEmail';
+export const FIELD_DOMAIN = 'domain';
+export const FIELD_FIRSTNAME = 'firstName';
+export const FIELD_LASTNAME = 'lastName';
+export const FIELD_IS_ADMIN = 'isAdmin';
```

Comment:
OTOH I think we should surface the opportunity to users to make a mailbox an admin since this is available in the CPs i.e. both Google and Professional Email